src/Uno.UWP/ApplicationModel/Appointments/AppointmentOrganizer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.ApplicationModel.Appointments
+{
+	public partial class AppointmentOrganizer : IAppointmentParticipant
+	{
+		public string DisplayName;
+		public string Address;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5452284)